extensions/amp-analytics/0.1/requests.js

```diff
@@ -222,6 +200,30 @@ export class RequestHandler {
     }
   }
 
+  /**
+   * @param {string} url
+   * @param {!ExpansionOptions} expansionOptions
+   * @param {!Object<string, (ResolverReturnDef|!SyncResolverDef)>=} bindings
+   * @return {!Promise<string>}
+   */
+  expandTemplateUrl_(url, expansionOptions, bindings) {
+    return this.variableService_
+      .expandTemplate(
+        url,
+        expansionOptions,
+        this.element_,
+        bindings,
+        this.allowlist_
+      )
+      .then((url) =>
+        this.urlReplacementService_
+          .expandUrlAsync(url, bindings, this.allowlist_)
+          .catch((e) =>
+            userAssert(false, `Could not expand URL "${url}": ${e.message}`)
+          )
```

Comment:
For an empty `requestOrigin`, will parsing with the native `URL()` trigger this catch block and thus stop the request from firing?.

Reply:
We don't get to this point if `requestOrigin` is empty or unset.

Reply:
SG! Thanks for the confirmation